CONTRIBUTING.rst

```diff
@@ -52,6 +52,14 @@ is pushed. A new tag can be pushed with::
 
     pipx run --spec="zest.releaser[recommended]>=6.22.1" fullrelease
 
+You can also run the zest.releaser process manually:
+
+1. pick a new version number!
+2. update the latest version and release date in ``docs/source/NEWS.rst``.
+3. update the ``__version__ = "{version}"`` in ``ldaptor/__init__.py``.
+4. tag the new release ``git tag v{version} -m 'Tagging {version}'``
+5. apply steps 2. through 3. for the development release version.
+
```

Comment:
```suggestion
The PyPI package publishing is done automatically via GitHub actions when a new tag is pushed.

To manually trigger the release for new version, follow this process:

* Create a new branch with the name `release-VER` (ex release-20.1.0) No need to create a separate issue.
* On that branch update the version from `ldaptor/__init__.py` to the
  final version.
* Update the NEWS file with the final version and release date.
* Push the commit and record the commit SHA as this will be the one used to
  tag and do the release.
* Include the tagged commit to the PR description.
* On the same branch, update the `ldaptor/__init__.py` to a `.dev0` version
  and update NEWS to a new empty unreleased section.
* Push the latest commit and request a review for the PR.
* Once the PR is approved, you can merge the PR
* After the merge, push the release tag. GitHub action will take care of the release.

You can check the GitHub actions release process here https://github.com/twisted/ldaptor/actions

Below is the template text for the unreleased NEWS section::

    20.0.1 (unreleased)
    -------------------

    Changes
    - Nothing changed yet.

The above process can be automated by running::

    pipx run --spec="zest.releaser[recommended]>=6.22.1" fullrelease
```